### PR TITLE
Fix output message

### PR DESCRIPTION
**Pull request by adybbroe**
This PR fixes the output message.

 - [ ] Closes #xxxx 
 - [ ] Tests added 
 - [x] Tests passed: Passes ``pytest`` 
 - [ ] Passes ``flake8`` 
 - [ ] Fully documented

### CODE REVIEW

File: activefires_pp/tests/test_geojson_utils.py [hunk 1]
```diff
@@ -124,13 +124,18 @@ def test_get_recent_geojson_files(fake_past_detections_dir):
     pattern = "sos_{start_time:%Y%m%d_%H%M%S}_{id:d}.geojson"
 
     recent = get_recent_geojson_files(fake_past_detections_dir, pattern, (starttime, endtime))
-    assert recent == ['sos_20210618_124819_0.geojson', 'sos_20210619_000651_1.geojson', 'sos_20210619_000651_0.geojson']
+    assert len(recent) == 3
+    assert recent[0] == 'sos_20210618_124819_0.geojson'
+    assert set(recent[1:3]) == set(['sos_20210619_000651_1.geojson', 'sos_20210619_000651_0.geojson'])
```
Comment (mraspaud) [inline activefires_pp/tests/test_geojson_utils.py:129]:
```suggestion
    assert set(recent) == {'sos_20210618_124819_0.geojson', 'sos_20210619_000651_1.geojson', 'sos_20210619_000651_0.geojson'}
```

File: activefires_pp/tests/test_geojson_utils.py [hunk 2]
```diff
@@ -124,13 +124,18 @@ def test_get_recent_geojson_files(fake_past_detections_dir):
...
     starttime = datetime(2021, 6, 18, 12, 0)
     endtime = datetime(2021, 6, 19, 12, 0)
     recent = get_recent_geojson_files(fake_past_detections_dir, pattern, (starttime, endtime))
-    assert recent == ['sos_20210618_124819_0.geojson', 'sos_20210619_000651_1.geojson',
-                      'sos_20210619_000651_0.geojson', 'sos_20210619_005803_0.geojson']
+
+    assert len(recent) == 4
+    assert recent[0] == 'sos_20210618_124819_0.geojson'
+    assert recent[-1] == 'sos_20210619_005803_0.geojson'
+    assert set(recent[1:3]) == set(['sos_20210619_000651_1.geojson', 'sos_20210619_000651_0.geojson'])
```
Comment (mraspaud) [inline activefires_pp/tests/test_geojson_utils.py:138]:
```suggestion
    assert set(recent) == ['sos_20210618_124819_0.geojson', 'sos_20210619_000651_1.geojson',
                           'sos_20210619_000651_0.geojson', 'sos_20210619_005803_0.geojson'}
```

**Review by mraspaud — COMMENTED** [review]
Just two small suggestions, looks good otherwise

File: activefires_pp/tests/test_geojson_utils.py (hunk 1 above)
Comment (adybbroe) [reply activefires_pp/tests/test_geojson_utils.py]: Off course, many thanks for spotting it.

File: activefires_pp/tests/test_geojson_utils.py (hunk 2 above)
Comment (adybbroe) [reply activefires_pp/tests/test_geojson_utils.py]: Thx, fixed

**Review by mraspaud — APPROVED** [review]
LGTM

**Pull request merged**